### DISCUSSION TITLE
Is the Colab running 2.1 now?

### DISCUSSION THREAD

**Discussion by illumnat** [TheLastBen/fast-stable-diffusion#936] [Q&A] [answered]
Quick question... The pulldown just shows V2-768. Is this the 2.1 model or still just the 2.0?

Thanks!

**Comment by illumnat** [comment, marked as answer]
Nevermind... just peeked under the table (i.e. hit "show code" on the colab) and saw `https://USER:{token}@huggingface.co/stabilityai/stable-diffusion-2-1` in there. ;-)

Thanks!

**Reply by TheLastBen** [reply]
https://github.com/TheLastBen/fast-stable-diffusion/commit/d1890736308bb1075fa4d59dc66963e0ef72cb6a
